pkg/project/project.go

```diff
@@ -1,13 +1,18 @@
 package project
 
 var (
-	description        = "The aws-operator handles Kubernetes clusters running on a Kubernetes cluster inside of AWS."
-	gitSHA             = "n/a"
-	name        string = "aws-operator"
-	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "n/a"
+	bundleVersion        = "5.5.0"
+	description          = "The aws-operator handles Kubernetes clusters running on a Kubernetes cluster inside of AWS."
+	gitSHA               = "n/a"
+	name          string = "aws-operator"
+	source        string = "https://github.com/giantswarm/aws-operator"
+	version              = "n/a"
 )
 
+func BundleVersion() string {
```

Comment:
This is taken from `master` branch. I'm not happy with the name. But we will get rid of version bundle concept later.